tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -26,7 +28,7 @@ def pipeline():
     return Pipeline.create(
         task="text_generation",
         model_path="hf:mgoin/TinyStories-1M-deepsparse",
-        engine_type="onnxruntime",
+        engine_kwargs={"engine_type": "onnxruntime"},
```

Comment:
Note: this diff shows that the UI for creating pipelines between V1 and V2 will change slightly as a result of the migration. Shouldn't we standardise it?

Reply:
Which UI change? `engine_kwargs` should have the same values as what the Pipeline `kwargs` currently have. Unless there's a specific parameter you're referring to?